CHANGELOG.md

```diff
@@ -25,6 +25,7 @@ All notable changes to this project will be documented in this file based on the
 * Launch tests with PHP 7.4
 * Added `nullable_type_declaration_for_default_null_value`, `no_alias_functions` CS rules [#1706](https://github.com/ruflin/Elastica/pull/1706)
 * Configured `visibility_required` CS rule for constants [#1723](https://github.com/ruflin/Elastica/pull/1723)
+* Added `Collapse` DSL to `QueryBuilder`
```

Comment:
Just a minor comment, but it would be perfect if you add the link to the PR ;)

Reply:
Done, thanks :)